app/src/main/java/com/onelist/dto/ShoppingList.kt

```diff
@@ -9,7 +9,12 @@ package com.onelist.dto
  * @property itemIDs the IDs of of the line items associated with the list
  * @constructor Creates a shopping list with ID, name, and user ID
  */
-data class ShoppingList(var listID: String, var name: String, var userIDs: List<String>, var itemIDs: List<String>) {
```

Comment:
This makes readability much better.

Reply:
Formatting can, and should, be performed automatically, and consistently, by the IDE.  Formatting code is really programmer's preference anyway, and doesn't necessarily decrease technical debt.